### PR TITLE
chore: update build workflow

### PR DESCRIPTION
**Pull request by Rory-Z**

### CODE REVIEW

File: .github/workflows/build_packages.yaml [hunk 1]
```diff
@@ -14,13 +18,31 @@ jobs:
...
         - [mips, mips-linux-gnu-gcc, mips-linux-gnu-g++]
         - [riscv64, riscv64-linux-gnu-gcc, riscv64-linux-gnu-g++]
         - [armhf, arm-linux-gnueabihf-gcc, arm-linux-gnueabihf-g++]
         - [armel, arm-linux-gnueabi-gcc, arm-linux-gnueabi-g++]
+        package_type:
+        - rpm
+        - deb
+        build_type:
+        - ""
+        - "-rqlite"
```
Comment (alvin1221) [inline .github/workflows/build_packages.yaml:32]: sqlite

File: .github/workflows/build_packages.yaml [hunk 2]
```diff
@@ -81,7 +87,19 @@ jobs:
...
+            if [ "$arch" == "arm64" ]; then
+              make_flags+=" -DCMAKE_TARGET_ARCHITECTURE=arm64 "
+              make_flags+=" -DGNU_MACHINE=aarch64-linux-gnu "
+              make_flags+=" toolchains $PWD/nng/extern/msquic/cmake/toolchains/aarch64-linux.cmake "
+            elif [ "$arch" == "armhf" ]; then
+              make_flags+=" -DCMAKE_TARGET_ARCHITECTURE=arm "
+              make_flags+=" toolchains $PWD/nng/extern/msquic/cmake/toolchains/arm-linux.cmake "
+              make_flags+=" -DGNU_MACHINE=arm-linux-gnueabihf "
+            fi
+            make_flags+=" -DONEBRANCH=1 "
```
Comment (Rory-Z) [inline .github/workflows/build_packages.yaml:101]: Can this line be moved below line 91 to make it look more friendly?

File: .github/workflows/build_packages.yaml [hunk 3]
```diff
@@ -81,7 +87,19 @@ jobs:
           make_flags+=" -DNNG_ENABLE_SQLITE=ON"
         fi
         if [ "${{ matrix.build_type }}" = "-msquic" ]; then
-          make_flags+=" -DNNG_ENABLE_QUIC=ON"
+          make_flags+=" -DNNG_ENABLE_QUIC=ON" 
+          if [ "$arch" != "x86_64" ]; then
```
Comment (Rory-Z) [inline .github/workflows/build_packages.yaml:91]: what happened if arch equal “amd64” ？

**Comment by Rory-Z** [comment]
Do we have any way to check “make_flags” is correct ？

File: .github/workflows/build_packages.yaml (hunk 3 above)
Comment (alvin1221) [reply .github/workflows/build_packages.yaml:91]: Sorry, my mistake

**Comment by alvin1221** [comment]
> 

It would be built fail if incorrect;

**Review by JaylinYu — APPROVED** [review]
not done yet

**Comment by JaylinYu** [comment]
merged due to cooperate commands

**Pull request merged**